app/models/keyword.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
-class Keyword
-  include ActiveModel::API
-
-  attr_accessor :id, :name
+class Keyword < ApplicationRecord
+  validates :name, presence: true
+  belongs_to :user, inverse_of: :keywords
```

Comment:
The user model is missing the association. So if we manipulate a keyword instance, we will get:

```
ActiveRecord::InverseOfAssociationNotFoundError: Could not find the inverse association for user (:keywords in User)
from /Users/xavier/.rvm/gems/ruby-3.0.1@google_search_ruby/gems/activerecord-7.0.1/lib/active_record/reflection.rb:236:in `check_validity_of_inverse!'
```